gradle/dependencies.gradle

```diff
@@ -21,7 +21,7 @@ ext {
       mapboxEvents              : '6.2.2',
       mapboxCore                : '3.1.1',
       mapboxNavigator           : "${mapboxNavigatorVersion}",
-      mapboxCommonNative        : '10.0.0-beta.12',
+      mapboxCommonNative        : '10.0.2',
```

Comment:
Noting that this brings OkHttp `4.9.0` so we should revamp the discussion around sunsetting the legacy Retrofit from `mapbox-java` and fix https://github.com/mapbox/mapbox-navigation-android/issues/3390 and https://github.com/mapbox/mapbox-navigation-android/issues/2790

cc @kmadsen

Reply:
> sunsetting the legacy Retrofit 

What does this mean? Rewrite mapbox-java without Retrofit?

Reply:
Nah, use the latest version of Retrofit that integrates / is based on OkHttp `4.x`.